### ISSUE TITLE
Keywords and AND in Run keyword if -keyword may note be detected properly

### ISSUE THREAD

**Issue by datakurre** [robotframework/pygmentslexer#13]
![screenshot_2019-01-14](https://user-images.githubusercontent.com/160447/51115661-94b65a00-1811-11e9-9291-94531eeb369e.png)

**Comment by pekkaklarck** [comment]
True. Probably not exactly trivial to fix, especially if we want to highlight special arguments like `AND` and `ELSE`, but worth taking a look. If it gets very complicated, may be better to spent the efforts for our new parser project that is about to start as well.

**Comment by datakurre** [comment]
I agree the parser is more important than this :)

**Comment by HelioGuilherme66** [comment]
@datakurre @pekkaklarck This issue will be fixed with PR #14, where control marked will be colorized as Token.Name.Function.Magic which is the same as Keywords (Token.Name.Function). The color must be actually changed in the styles that support Token.Name.Function.Magic. Known styles are: coffee, material, solarized-dark and solarized-light.
